test/src/sender/main.cpp

```diff
@@ -23,119 +23,164 @@
+        typeid(outp_data),typeid(outp_pack),typeid(outp_ack)
+    };
+    cadmium::dynamic::modeling::Models submodels_TOP = {
+        generator_con, generator_ack, sender1
+    };
+    cadmium::dynamic::modeling::EICs eics_TOP = {};
+    cadmium::dynamic::modeling::EOCs eocs_TOP = {
+        cadmium::dynamic::translate::make_EOC
+            <sender_defs::packet_sent_out,outp_pack>("sender1"),
+            cadmium::dynamic::translate::make_EOC
+                <sender_defs::ack_received_out,outp_ack>("sender1"),
+                cadmium::dynamic::translate::make_EOC
+                    <sender_defs::data_out,outp_data>("sender1")
+    };
+    cadmium::dynamic::modeling::ICs ics_TOP = {
+        cadmium::dynamic::translate::make_IC
+            <iestream_input_defs<Message_t>::out,
+                sender_defs::control_in>("generator_con","sender1"),
+            cadmium::dynamic::translate::make_IC
+                <iestream_input_defs<Message_t>::out,
+                    sender_defs::ack_in>("generator_ack","sender1")
+    };
+    std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> TOP =
+        std::make_shared<cadmium::dynamic::modeling::coupled<TIME>>(
+            "TOP",submodels_TOP,iports_TOP,
+                oports_TOP,eics_TOP,eocs_TOP,ics_TOP);
 
+    ///****************////
 
-/************************/
-/*******TOP MODEL********/
-/************************/
-cadmium::dynamic::modeling::Ports iports_TOP = {};
-cadmium::dynamic::modeling::Ports oports_TOP = {typeid(outp_data),typeid(outp_pack),typeid(outp_ack)};
-cadmium::dynamic::modeling::Models submodels_TOP = {generator_con, generator_ack, sender1};
-cadmium::dynamic::modeling::EICs eics_TOP = {};
-cadmium::dynamic::modeling::EOCs eocs_TOP = {
-  cadmium::dynamic::translate::make_EOC<sender_defs::packet_sent_out,outp_pack>("sender1"),
-cadmium::dynamic::translate::make_EOC<sender_defs::ack_received_out,outp_ack>("sender1"),
-cadmium::dynamic::translate::make_EOC<sender_defs::data_out,outp_data>("sender1")
-};
-cadmium::dynamic::modeling::ICs ics_TOP = {
-  cadmium::dynamic::translate::make_IC<iestream_input_defs<Message_t>::out,sender_defs::control_in>("generator_con","sender1"),
-  cadmium::dynamic::translate::make_IC<iestream_input_defs<Message_t>::out,sender_defs::ack_in>("generator_ack","sender1")
-};
-std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> TOP = std::make_shared<cadmium::dynamic::modeling::coupled<TIME>>(
- "TOP", 
- submodels_TOP, 
- iports_TOP, 
- oports_TOP, 
- eics_TOP, 
- eocs_TOP, 
- ics_TOP 
-  );
-
-///****************////
-
-    auto elapsed1 = std::chrono::duration_cast<std::chrono::duration<double, std::ratio<1>>>(hclock::now() - start).count();
-    cout << "Model Created. Elapsed time: " << elapsed1 << "sec" << endl;
+    auto elapsed1 = std::chrono::duration_cast<std::chrono::duration
+        <double,std::ratio<1>>> (hclock::now() - start).count();
+    cout <<"Model Created. Elapsed time: "<<elapsed1<<"sec"<< endl;
```

Comment:
No space is required between cout and <<. 
It can be removed

Reply:
solved

Reply:
There is space between << and endl. It can be removed.

Reply:
solved